compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3460,7 +3460,9 @@ object Parsers {
             else ident().toTypeName
           val hkparams = typeParamClauseOpt(ParamOwner.Hk)
           val bounds =
-            if paramOwner.acceptsCtxBounds then typeAndCtxBounds(name) else typeBounds()
+            if paramOwner.acceptsCtxBounds then typeAndCtxBounds(name)
+            else if in.featureEnabled(Feature.modularity) && paramOwner == ParamOwner.Type then typeAndCtxBounds(name)
```

Comment:
why does this need the modularity import? isn't it part of SIP-64 so should be enabled without a feature?

Reply:
Hmmm, I assumed that, since it was meant to be pushed to the next minor, it should be under a language import.
If it's not, can we still do another RC/backport?
@WojciechMazur

Reply:
Yes, we should fix that in the followup and backport to RC2.
This PR was explicitly marked as one of the things that should be included in 3.6